static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-epic-testimonials-group.js

```diff
@@ -8,7 +8,7 @@ import { acquisitionsTestimonialBlockTemplate } from 'common/modules/commercial/
 const getTestimonialText = () => {
     const mvtId = getMvtValue();
     const numberOfViews = overallNumberOfViews();
-    return testimonialCycleGroup[mvtId % 8 + numberOfViews % 8];
+    return testimonialCycleGroup[(mvtId + numberOfViews) % 8];
```

Comment:
Just a thought: if this variant wins and this testimonial cycle thing survives, you could replace `% 8` with `% testimonialCycleGroup.length` which would guarantee staying within the array bounds even if someone unaware of this code adds elements to the array